qiskit/transpiler/passes/calibration/rzx_builder.py

```diff
@@ -343,36 +332,64 @@ def _filter_comp_tone(time_inst_tup):
     return False
 
 
-def _check_calibration_type(cx_sched) -> Tuple[CXCalType, List[Play], List[Play]]:
+def _check_calibration_type(
+    inst_sched_map: InstructionScheduleMap, qubits: List[int]
+) -> Tuple[CRCalType, List[Play], List[Play]]:
     """A helper function to check type of CR calibration.
 
     Args:
-        cx_sched: A target schedule to stretch.
+        inst_sched_map: instruction schedule map of the backends
+        qubits: ordered tuple of qubits for cross resonance (q_control, q_target)
 
     Returns:
         Filtered instructions and most-likely type of calibration.
 
     Raises:
         QiskitError: Unknown calibration type is detected.
     """
+    cal_type = None
+    if "cx" in inst_sched_map.qubit_instructions(qubits):
+        cr_sched = inst_sched_map.get("cx", qubits=qubits)
+    elif "ecr" in inst_sched_map.qubit_instructions(qubits):
+        cr_sched = inst_sched_map.get("ecr", qubits=qubits)
+        cal_type = CRCalType.ECR_FORWARD
```

Comment:
It assumes reverse ECR gate will be never provided by the device. Just a curiosity, is the reverse ECR not locally equivalent to the forward ECR, or new backend intentionally avoids fake bidirectional coupling? If latter is true, we might need to modify the logic of ECR_CX (at least we need to support the case that the forward CX is missing in the instmap).

Reply:
From my understanding that is correct, only forward ECR is provided. While they are locally equivalent, the durations of the forward and reverse ECR would be different (if implemented with the same control/target channels for both) or the fidelities of the forward and reverse ECR would vary (if implemented in the different cross resonance configurations). I believe the direction with better fidelity is provided, and the fact that durations are equal lend it better to error mitigation techniques.